x-pack/legacy/plugins/alerting/server/routes/create.test.ts

```diff
@@ -30,6 +31,17 @@ const mockedAlert = {
   ],
 };
 
+const mockedAlertComputerFields = {
```

Comment:
`Computer` fields?  I was thinking maybe that was a mistype of `Computed`, but that doesn't seem quite right either ...

Reply:
Yeah, was meant to be computed... inlined the variable, it isn't really needed as a stand alone